src/diffusers/loaders/single_file_utils.py

```diff
@@ -57,6 +57,9 @@
 
 logger = logging.get_logger(__name__)  # pylint: disable=invalid-name
 
+# Set global timeout
```

Comment:
Can we actually define this as a constant `DIFFUSERS_REQUEST_TIMEOUT` in this file
https://github.com/huggingface/diffusers/blob/de6a88c2d7659c616b44c0856677335110b8ff2e/src/diffusers/utils/constants.py#L4

And then import it from constatnts when needed.

Reply:
Resolved by: [44b5c60](https://github.com/huggingface/diffusers/pull/11156/commits/44b5c6091c4f11d9bf8f15bd805ea6d496b7cfe6)